readthedocs/builds/constants_docker.py

```diff
@@ -75,7 +76,7 @@
 # Set latest aliases for OS and tools.
 _OS = RTD_DOCKER_BUILD_SETTINGS["os"]
 _TOOLS = RTD_DOCKER_BUILD_SETTINGS["tools"]
-_OS["ubuntu-lts-latest"] = _OS["ubuntu-22.04"]
+_OS["ubuntu-lts-latest"] = _OS["ubuntu-24.04"]
```

Comment:
Should we wait to bump this until we ship and test 24.04 is working properly?

Reply:
Yes. Good point 👍